base/exports.jl

```diff
@@ -1148,7 +1148,6 @@ export
     ismarked,
     isopen,
     isreadonly,
-    limit_output,
```

Comment:
I think it was only for a few months in 0.5-dev, when I was experimenting with whether `limit_output(io)` would be more concise than `get(io, :limit, false) === true` (before we just went with the `get` form consistently), so it should be OK to just drop it